client/my-sites/domains/transfer-domain/index.jsx

```diff
@@ -77,6 +77,12 @@ export class TransferDomain extends Component {
 			} )
 		);
 
+		upgradesActions.addItem(
```

Comment:
You can use `addItems` and do it in a single call. I think it's better as it initiates only one API call then.